backend/maelstro/config/config.py

```diff
@@ -77,6 +77,30 @@ def has_db_logging(self) -> bool:
     def get_db_config(self) -> DbConfig:
         return DbConfig(**self.config.get("db_logging", {}))
 
+    def get_transformations(self) -> dict[str, Any]:
+        return self.config.get("transformations", {})  # type: ignore
+
+    def get_transformation_pair(self, src: str, dst: str) -> list[dict[str, Any]]:
+        return self.get_transformation_pairs()[f"{src} -> {dst}"]  # type: ignore
```

Comment:
Seems you will get a keyError if there is no transformations

Reply:
good catch, added empty list as default value